### ISSUE TITLE
awot_time_series_flight_data.ipynb

### ISSUE THREAD

**Issue by cjjhnst2** [nguy/AWOT#50]
Hello, I am trying to plot a flight map and a time series of variables and am running into this error:

```
# Load the needed packages
import os
import matplotlib.pyplot as plt 
import matplotlib as mpl # in python
import numpy as np
import matplotlib.colors as mcolors
import datetime
from netCDF4 import Dataset
from matplotlib.ticker import MultipleLocator, FormatStrFormatter
from matplotlib import rcParams, colors, cm
from awot.graph.common import create_basemap


import awot
from awot.graph import FlightLevel, RadarVerticalPlot, RadarUtilityPlot

%matplotlib inline
```

```
# Set the project name
Project="OWLES"
# Set the path for data file
flname = "/data/keeling/a/cjjhnst2/a/data/UWYO_king_air/kadata/20140107a.c25.nc"

proj = 'cea'
Wbarb_Spacing = 300 # Spacing of wind barbs along flight path (sec)

# Choose the X-axis time step (in seconds) where major labels will be
XlabStride = 3600

# Should landmarks be plotted? [If yes, then modify the section below
Lmarks=True

# Optional variables that can be included with AWOT
# Start and end times for track in Datetime instance format
start_time = "2014-01-07 13:00:00"
end_time = "2014-01-07 16:45:00"

corners = [-79.,42.,-74.,44.]

fl = awot.io.read_netcdf(fname=flname, platform='uwka')

fig, ax = plt.subplots(1, 1, figsize=(9, 9))

bm = create_basemap(corners=corners, proj=proj, resolution='l', area_thresh=1.,ax=ax)

# Instantiate the Flight plotting routines
flp = FlightLevel(fl, basemap=bm)
flp.plot_trackmap(
#                 start_time=start_time, end_time=end_time,
                 color_by_altitude=True, track_cmap='spectral',
                 min_altitude=50., max_altitude= 4000.,
                 addlegend=True, addtitle=True)
```
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-261d6e5e07d1> in <module>()
      4 
      5 # Instantiate the Flight plotting routines
----> 6 flp = FlightLevel(fl, basemap=bm)
      7 flp.plot_trackmap(
      8 #                 start_time=start_time, end_time=end_time,

/data/keeling/a/cjjhnst2/anaconda2/lib/python2.7/site-packages/awot-0.2.12-py2.7.egg/awot/graph/flight_level.pyc in __init__(self, flightdata, basemap, lon_name, lat_name, alt_name, time_name, uwind_name, vwind_name)
    105             # Calculate x,y map position coordinates
    106             self.x, self.y = self.basemap(
--> 107                 self.longitude['data'][:], self.latitude['data'][:])
    108 
    109 #################

TypeError: 'NoneType' object has no attribute '__getitem__'

```
Any ideas to what might be going on?

**Comment by nguy** [comment]
If this is still an issue, can you put a print statement or such making sure that the basemap, lon, and lats are all defined?

**Comment by nguy** [comment]
Fixed #51
